projects/angular/src/data/datagrid/render/main-renderer.ts

```diff
@@ -287,4 +289,14 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, Af
       this.columnsSizesStable = true;
     }
   }
+
+  private setVisibility() {
+    if (this.el.nativeElement.offsetParent) {
+      this.isVisible = true;
+      this.datagrid.datagrid.nativeElement.style.visibility = 'visible';
```

Comment:
Rename to `clrDatagridComponent`. Accessing it through `this.datagrid.datagrid` feels odd.

```suggestion
      this.clrDatagridComponent.datagrid.nativeElement.style.visibility = 'visible';
```

Reply:
I don't think the `clr` prefix is needed.